packages/frontend-2/components/settings/workspaces/members/GuestsTable.vue

```diff
@@ -84,6 +84,17 @@
       :user="userToModify"
       :workspace-id="workspaceId"
     />
+
+    <SettingsSharedChangeRoleDialog
+      v-model:open="showChangeUserRoleDialog"
+      :name="userToModify?.user.name ?? ''"
+      :is-workspace-admin="isWorkspaceAdmin"
```

Comment:
Same with this one, it doesn't exist in `SettingsSharedChangeRoleDialog` however we probably should add some logic for this, because you can only make someone an admin when you are one